### DISCUSSION TITLE
How to in papers server use the easy mob?

### DISCUSSION THREAD

**Discussion by qazwsxedc285** [maruohon/litematica#668] [Q&A]
I in 1.19.3 paper server playing i use the easy mob i can't but the block down. Have show the Action prevented by the easy mobs how to Solve the problem?

**Comment by maruohon** [comment]
Easy Place mode won't be able to rotate any blocks on a server that doesn't handle the "accurate placement protocol". But if you set the `easyPlaceProtocolVersion` option to `None` or `Slabs Only`, then you could still use it to place most blocks that don't have a rotation. But if the server prevents clicking on air blocks, like many Spigot/Paper servers/plugins do, then you probably can't use Easy Place mode at all.

However the "action prevented by Easy Place mode" warning only happens when the mod prevents the right click in the first place. That only happens if you try to place the wrong block in the wrong place, or if you try to spam click to place the same block many times within a short time period.
